Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/goswagger/swagger:0.15.0
+FROM quay.io/goswagger/swagger:v0.17.0
```

Comment:
why not 18?

Reply:
because currently latest is aligned with v0.17.0 in https://quay.io/repository/goswagger/swagger?tab=tags, so I assumed it would be better to take that version (maybe they consider it less stable). Anyway, I posted a question on that:
https://github.com/go-swagger/go-swagger/issues/1873

Reply:
I'll assume for now that they simply forgot to update latest.

Reply:
Please updated commit (version changed to v0.18.0.

Reply:
back to v0.17.0